src/content/snapshotOptions/animations.mdx

```diff
@@ -105,6 +109,62 @@ MotionGlobalConfig.skipAnimations = isChromatic();
 
 </div>
 
+### Playwright and Cypress
+
+When using Playwright or Cypress, you can assert that specific elements are visible in the DOM to confirm an animation has finished. Alternatively, use `wait` or `waitForTimeout` functions to wait for the animation to complete.
+
+Another strategy is to inject a variable into the window object to disable animations. For example, in your E2E test, set the `disableAnimations` property on the `window` object to `true`.
+
+{/* prettier-ignore-start */}
+
+<IntegrationSnippets>
+  <Fragment slot="playwright">
+    ```ts title="tests/Product.spec.js|ts"
+    import { test, expect } from "@chromatic-com/playwright";
+
+    test.describe("Products Page", () => {
+      test("Successfully loads the page", async ({ page }) => {
+        // Set a property on the the window object to disable animations.
+        await page.addInitScript(() => {
+          window.disableAnimations = true
+        });
+
+        await page.goto("/products");
+        await expect(page.getByText("Product details")).toBeVisible();
+      });
+    });
+    ```
+  </Fragment>
+  <Fragment slot="cypress">
+    ```ts title="cypress/e2e/Product.cy.js|ts"
+    describe("Not found", () => {
+        // Set a property on the the window object to disable animations.
+        Cypress.on('window:before:load', (win) => {
+          win.disableAnimations = true
+        })
+
+      it("Successfully loads the page", { env: {
```

Comment:
@winkerVSbecks, so that we have some consistency across the examples for Playwright and Cypress, are you ok with either:
- Remove the `pauseAnimationAtEnd` option from the Cypress example
- Or, if the intention is to pause the animations at the end for both examples, add it to the Playwright's example

I'll leave it up to you.